dfastmi/gui/dialog_view_model.py

```diff
@@ -87,7 +95,8 @@ def output_dir(self):
     @output_dir.setter
     def output_dir(self, value):
         self._output_dir = value
-        self.model.section["OutputDir"] = value
```

Comment:
Why were all the extra checks added? We know that the view model has a `model` - it is never None, so no need to check it everywhere.

Reply:
good idea! will update